gulpfile.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  'use strict';
+
+  var gulp = require('gulp'),
+      jscs = require('gulp-jscs'),
+      jshint = require('gulp-jshint'),
```

Comment:
Have you ever played with eslint? I’ve been using that for a while, but haven’t looked if it has any style-specific rules.